mmv1/products/dialogflowcx/Environment.yaml

```diff
@@ -61,9 +60,15 @@ examples:
     primary_resource_id: 'development'
     vars:
       agent_name: 'dialogflowcx-agent'
+  - !ruby/object:Provider::Terraform::Examples
+    skip_docs: true
+    name: 'dialogflowcx_environment_regional'
+    primary_resource_id: 'development'
+    vars:
+      agent_name: 'issue12880'
 skip_sweeper: true
 id_format: '{{parent}}/environments/{{name}}'
-import_format: ['{{parent}}/environments/{{name}}']
+import_format: ['{{%parent}}/environments/{{name}}']
```

Comment:
It looks like the old implementation supported a short format `{{parent}}/{{name}}`. Could we add format as well so that we don't break existing workflows?

Reply:
Good point, I wasn't being particularly careful here (I didn't even delete the old code template!). Reverted this set of changes, hopefully the generated code goes back to how it was.

Reply:
I'm marking this as "resolved" because there's no longer generated diffs in `resourceDialogflowCXVersionImport` + `resourceDialogflowCXEnvironmentImport`